tests/dummy/app/styles/app.scss

```diff
@@ -201,6 +201,16 @@ th, td {
   color: #bbb;
 }
 
+pre > code {
+  border: none;
+  margin: 0;
+  padding: 0 .5rem;
+}
+
+:not(pre) > code[class*="language-"], pre[class*="language-"] {
+  background-color: #f8f8f8;
+}
+
```

Comment:
Is there more constrained scoping we could do here, maybe a class applied to the `CodeBlock` component?

Reply:
Updated to use a class for styling the `code` element, but the `CodeBlock` doesn't provide a way to add a class to the `pre` directly. Instead, I updated the background color of the `.code-template-toggle` to match the prism theme, rather than updating the theme's background color.

The "after" screenshot in the PR description shows how this looks.